spec/wasabi/document/escaped_endpoint_spec.rb

```diff
@@ -0,0 +1,17 @@
+
+require "spec_helper"
+
+describe Wasabi::Document do
+  context "with: escaped_endpoint.wsdl" do
+
+    subject(:document) { Wasabi::Document.new fixture(:escaped_endpoint).read }
+
+    describe '#endpoint' do
+      subject { super().endpoint }
```

Comment:
```suggestion
```

I would expect _this change_ to do change nothing, and simplify.

Reply:
yeah, that is true 👍